3rdParty/velocypack/include/velocypack/velocypack-common.h

```diff
@@ -245,11 +356,11 @@ static inline uint64_t readUInt64(uint8_t const* start) noexcept {
 }
 
 static inline void storeUInt64(uint8_t* start, uint64_t value) noexcept {
-  uint8_t const* end = start + 8;
-  do {
-    *start++ = static_cast<uint8_t>(value & 0xffU);
-    value >>= 8;
-  } while (start < end);
+ if constexpr (!isLittleEndian()) {
+   value = hostToLittle(value);
+ }
```

Comment:
```suggestion
 value = hostToLittle(value);
```
If we have the constexpr with the early return in `hostToLittle` this should compile down to a noop anyway.

Reply:
I prefer  to keep it clean to the reader that we do conversion only in big-endian case. An as it is constexpr it costs us nothing.

Reply:
IMHO this should be the job of `hostToLittle` (that's why I suggested the early return) and it should be relatively clear that on a little endian system the function should do nothing. So even though the `if` is more explicit, it also adds some complexity to the code that IMHO is not necessary (the same actually goes for the `if`s in the two functions above). But no strong opinion here - I'm fine with leaving it as is.